tool/mk_builtin_loader.rb

```diff
@@ -275,6 +275,7 @@ def generate_cexpr(ofile, lineno, line_file, body_lineno, text, locals, func_nam
     next unless Symbol === param
     next unless local_candidates.include?(param.to_s)
     f.puts "MAYBE_UNUSED(const VALUE) #{param} = rb_vm_lvar(ec, #{-3 - i});"
+    f.puts "MAYBE_UNUSED(VALUE *) #{param}__ptr = (VALUE *)&ec->cfp->ep[#{-3 - i}];"
```

Comment:
- constify `#{param}__ptr` itself.
- dereference `*#{param}__ptr` and no `MAYBE_UNUSED` on it.
- adding `rb_vm_lvar_ptr` or something would be better.

```suggestion
    f.puts "VALUE *const #{param}__ptr = (VALUE *)&ec->cfp->ep[#{-3 - i}];"
    f.puts "MAYBE_UNUSED(const VALUE) #{param} = *#{param}__ptr;"
```